deploy/crds.yaml

```diff
@@ -80,6 +86,16 @@ spec:
                   type: string
           status:
             type: object
+            properties:
+              state:
+                type: string
+                enum:
+                  - Pending
+                  - Adding
+                  - Added
+                  - Removing
+                  - Removed
+                default: Pending
```

Comment:
We should also add `additionalPrinterColumns` here to output the state in `kubectl` commands